docs/installation/configuration/cmake_options.rst

```diff
@@ -120,6 +120,12 @@ dependency on other options.
           and therefore the :ref:`persistence_service`.
         - ``ON`` ``OFF``
         - ``ON``
+    *   - :class:`APPEND_PROJECT_NAME_TO_INCLUDEDIR`
+        - When ``ON`` headers are installed to a path ending with a folder called ``fastrtps``.
+          This avoids include directory search order issues when overriding this package from a
```

Comment:
I'd force line breaks so that readers don't need to scroll right. This was also meant for the table to fit entirely in the page, but the new option name is large enough so that the table widens too much; correcting it would require adjusting all the line breaks (`|br|`) in all the previous entries
```suggestion
        - When ``ON`` headers are installed to a path ending with a folder called ``fastrtps``. |br|
          This avoids include directory search order issues when overriding this package from a |br|
```